src/main/java/dev/tycho/stonks/managers/DatabaseHelper.java

```diff
@@ -177,35 +177,35 @@ public void openCompanyMembers(Player player, String companyName) {
             } catch (SQLException e) {
               e.printStackTrace();
             }
-            return MemberListGui.getInventory(company, list);
+            return new MemberListGui(company, list);
           } catch (SQLException e) {
             e.printStackTrace();
           }
           return null;
         })
         .abortIfNull()
-        .sync((result) -> result.open(player))
+        .sync(gui -> gui.show(player))
         .execute();
   }
 
   public void openCompanyAccounts(Player player, String companyName) {
     Stonks.newChain()
         .asyncFirst(() -> {
-          sendMessage(player, "Loading accounts...");
+          player.sendMessage(ChatColor.AQUA + "Loading accounts...");
           try {
             Company company = databaseManager.getCompanyDao().getCompany(companyName);
             if (company == null) {
-              sendMessage(player, "That company doesn't exist!");
+              player.sendMessage(ChatColor.RED + "That company doesn't exist!");
```

Comment:
Revert this

Reply:
https://github.com/schlatt-co/stonks/blob/a34c87a8a16fdce21ba0462044aedb07a35af01b/src/main/java/dev/tycho/stonks/managers/DatabaseHelper.java#L197